src/lib/components/workflow-status.svelte

```diff
@@ -20,58 +19,23 @@
     Terminated: translate('workflows', 'terminated'),
     Paused: translate('workflows', 'paused'),
   };
-
-  const colors = {
-    Running: 'blue',
-    TimedOut: 'orange',
-    Completed: 'green',
-    Failed: 'red',
-    ContinuedAsNew: 'purple',
-    Canceled: 'yellow',
-    Terminated: 'gray',
-    Paused: 'yellow',
-  };
-
-  $: color = colors[status];
-  $: label = humanFriendlyNames[status];
-  $: isRunning = label === humanFriendlyNames.Running;
 </script>
 
 <span class="flex text-center text-sm font-medium leading-4">
```

Comment:
We could just use `<Badge />` here and get rid of those extra color styles

```
<Badge type={color}>
  <span class="whitespace-nowrap">{label}</span>
  {#if isRunning}
    <HeartBeat {delay} />
  {/if}
</Badge>
```

Reply:
Let's give it a shot.